### ISSUE TITLE
Holiday Event Error | It was totally me lol

### ISSUE THREAD

**Issue by Bam8647** [Monika-After-Story/MonikaModDev#4922]
Well I never delved into this part of the code but here we go ;P

![AAAA3](https://user-images.githubusercontent.com/34694679/67740189-f3387780-f9e2-11e9-8525-9e4c9e63c936.PNG)

**Comment by multimokia** [comment]
You have rpy files in your game directory.

Delete all rpy files and reinstall.

**Comment by Bam8647** [comment]
I've had rpy files in my directory the whole time I've been making my own topics and it's been running just fine..? I have a rpyc version of all the modded files so that they may update successfully. The only files that are having problems are event handlers and holidays. And they both successfully received the update.

**Comment by multimokia** [comment]
We don't offer support to those who use source code versions specifically for these reasons. Like Monika says, the game may not update correctly because `.rpy` files override `.rpyc`. So they were updated, and now they're downgraded again.

However, the addition of a new `.rpyc` file, in combination with a new event database added in it, with the event_handler being out of date, means that it's not capable of accepting the new db, hence why it thinks that it's `not of type dict`.

**Comment by Bam8647** [comment]
Okay, so If I convert all rpy files to rpyc and reinstall the mod, it should correct the issue? Also, you mention the creation of new rpyc files. Are you referring to my submodded folders containing the topics I created? If that's the case, then how might I continue to use my submodded topics without it's files interferring with the updates? And, yes the files I modded are named differently from the MAS files.
